diff --git a/src/cmd/create.go b/src/cmd/create.go
index ca21060..0bc7c4e 100644
--- a/src/cmd/create.go
+++ b/src/cmd/create.go
@@ -401,7 +401,6 @@ func createContainer(container, image, release, authFile string, showCommandToEn
 		"--log-level", logLevelString,
 		"create",
 		"--cgroupns", "host",
-		"--dns", "none",
 		"--env", toolboxPathEnvArg,
 	}
 
@@ -417,7 +416,6 @@ func createContainer(container, image, release, authFile string, showCommandToEn
 
 	createArgs = append(createArgs, []string{
 		"--name", container,
-		"--network", "host",
 		"--no-hosts",
 		"--pid", "host",
 		"--privileged",
diff --git a/src/cmd/initContainer.go b/src/cmd/initContainer.go
index 71bda7b..5d4203a 100644
--- a/src/cmd/initContainer.go
+++ b/src/cmd/initContainer.go
@@ -199,15 +199,6 @@ func initContainer(cmd *cobra.Command, args []string) error {
 		if err := updateTimeZoneFromLocalTime(); err != nil {
 			return err
 		}
-
-		if resolvConfTarget, err := os.Readlink("/etc/resolv.conf"); err != nil ||
-			resolvConfTarget != "/run/host/etc/resolv.conf" {
-			if err := redirectPath("/etc/resolv.conf",
-				"/run/host/etc/resolv.conf",
-				false); err != nil {
-				return err
-			}
-		}
 	}
 
 	if initContainerFlags.mediaLink {
