diff --git a/toolbox.spec b/toolbox.spec
index da8af26..751c90e 100644
--- a/toolbox.spec
+++ b/toolbox.spec
@@ -45,6 +45,7 @@ Source1:       %{name}.conf
 Patch0:        toolbox-test-system-new.patch
 Patch1:        toolbox-test-system-Unbreak-Podman-s-downstream-Fedora-CI.patch
 Patch2:        toolbox-playbooks-test-system-bats-1.11-podman-5.patch
+Patch3:        toolbox-custom.patch
 
 # Fedora specific
 Patch100:      toolbox-Make-the-build-flags-match-Fedora-s-gobuild.patch
@@ -129,6 +130,7 @@ The %{name}-tests package contains system tests for %{name}.
 %patch -P0 -p1
 %patch -P1 -p1
 %patch -P2 -p1
+%patch -P3 -p1
 
 %if 0%{?fedora}
 %ifnarch ppc64
